example/Makefile

```diff
@@ -34,9 +47,9 @@ opt_demo.ll: ir_demo.ll
 	$(OPT) -postdomtree -mem2reg -indvars -loop-simplify -branch-prob -scalar-evolution -S < $< > $@
 
 
-# ci based run of a demo prgram with multiple source files
+# ci-based run of a demo prgram with multiple source files
 ci_mult_files: ci_mult_files.ll $(CI_LIB)
-	$(CC) $(INC) -L$(CI_LIB_HOME) -Wl,-rpath=$(CI_LIB_HOME) -g $< -o $@ -lpthread -lci
+	$(CC) $(INC) -L$(CI_LIB_HOME) -Wl,-rpath,$(CI_LIB_HOME) -g $< -o $@ -lpthread -lci
```

Comment:
This won't run on linux. We need to pass these commands by setting a variable based on the system.

Reply:
GNU linker is able to handle both `-rpath=` and `-rpath,` syntaxes.